openhtf/io/http_api.py

```diff
@@ -119,14 +121,25 @@ def do_GET(self):  # pylint: disable=invalid-name
       """
       result = {'test': util.ConvertToBaseTypes(self.executor.GetState()),
                 'framework': util.ConvertToBaseTypes(self.executor)}
-      self.wfile.write(json.dumps(result))
+      with open(os.devnull, 'w') as f:
+        with contextlib.redirect_stderr(f):  # Don't spam the terminal.
```

Comment:
Good call. I traced this down at the time, but for some reason didn't think to override it and opted for the brute-force redirect instead. I'll fix it.